sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -541,9 +541,10 @@ node add(const device& syclDevice, T cgf, const property_list& propList = {});
 |This function adds a command group function object to a graph. The function
 object statically contains a group of commands, of which a single command is
 executed at runtime. A function object can be a host task which is scheduled by
```

Comment:
> A function object can be ...

It might be easier to just remove this sentence enumerating what the different function objects could be. We could also have a child graph from `handler::ext_oneapi_graph` but don't list that explicitly.